.eslintrc.json

```diff
@@ -24,7 +34,97 @@
     {
       "files": ["*.ts", "*.tsx"],
       "extends": ["plugin:@nx/typescript"],
-      "rules": {}
+      "rules": {
+        "testing-library/prefer-screen-queries": 0,
+        "comma-dangle": [
+          "error",
+          {
+            "arrays": "always-multiline",
+            "exports": "only-multiline",
+            "functions": "never",
+            "imports": "only-multiline",
+            "objects": "only-multiline"
+          }
+        ],
+        "eol-last": "error",
+        "import/order": [
+          "warn",
+          {
+            "groups": [
+              "builtin",
+              "external",
+              "index",
+              "sibling",
+              "parent",
+              "internal"
+            ]
+          }
+        ],
+        "jsx-quotes": ["warn", "prefer-double"],
+        "max-len": [
+          "warn",
+          {
+            "code": 120,
+            "ignorePattern": "d=\"([\\s\\S]*?)\""
```

Comment:
This was put in place instead of setting the `ignorePattern` at the top level for `Assets` like the original `.eslintrc.json`, it skips SVG path warnings. We could also place `"ignoreStrings": true` but this would affect copy text as well.